src/collective/fingerpointing/subscribers/registry_logger.py

```diff
@@ -23,6 +23,6 @@ def registry_logger(event):
 
         if IRecordModifiedEvent.providedBy(event):
             action = 'modify'
-            extras = 'object={0} value={1}'.format(event.record, event.record.value)
+            extras = u'object={0} value={1}'.format(event.record, event.record.value)
```

Comment:
I read some code related to ``site_title`` property and didn't find why  this property would be anything different than ``unicode``.

I also started a Plone 5 instance, set the title in the control panel, and look for it programmatically to see if it is ``unicode`` and it is:

```
Starting debugger (the name "app" is bound to the top-level Zope object)

>>> registry = app.Plone.portal_registry
>>> registry['plone.site_title']
u'\xdcml\xe4ut'
>>> print registry['plone.site_title']
Ümläut
```

Of course you can push programmatically any value into ``site_title``, but this is not a real use case.